### PR TITLE
Comment list color

### PR DESCRIPTION
**Pull request by penk85**
WPC-340: Highlight color on comment table reflects the status of the comment state.

### CODE REVIEW

**Review by HanaAyman — DISMISSED** [review]

**Comment by HanaAyman** [comment]
👍

File: packages/editor/src/client/global.less [hunk 1]
```diff
@@ -1,4 +1,32 @@
 .highlighted-row,
 .highlighted-row .rs-table-cell-group .rs-table-cell {
-    background-color: #f2faff;
+  background-color: #f2faff;
```
Comment (roosnic1) [inline packages/editor/src/client/global.less:3]: i think we should do the highlighted-row from the duplicating page and article the same way.

**Review by roosnic1 — CHANGES_REQUESTED** [review]
Looks awesome but i think we should do the same for the highlighted row.

File: packages/editor/src/client/global.less [hunk 2]
```diff
@@ -1,9 +1,9 @@
-.highlighted-row,
-.highlighted-row .rs-table-cell-group .rs-table-cell {
-  background-color: #f2faff;
-}
-
 .rs-table {
+  .highlighted-row,
+  .highlighted-row &-cell-group &-cell {
```
Comment (bishoymelek) [inline packages/editor/src/client/global.less:3]: Out of curiosity, what does &-cell-group means here? same as `.rs-table-cell-group` ?

**Review by bishoymelek — COMMENTED** [review]
Great work, small question

File: packages/editor/src/client/global.less (hunk 2 above)
Comment (penk85) [reply packages/editor/src/client/global.less:3]: @bishoymelek yes

**Review by bishoymelek — APPROVED** [review]
🎉

**Review by roosnic1 — APPROVED** [review]
looks awesome

**Pull request merged**